jetty-util/src/test/java/org/eclipse/jetty/util/HostPortTest.java

```diff
@@ -31,14 +31,10 @@ public static Stream<Arguments> validAuthorityProvider()
 
         return Stream.of(
             Arguments.of("", "", null),
-            Arguments.of(":80", "", "80"),
             Arguments.of("host", "host", null),
             Arguments.of("host:80", "host", "80"),
             Arguments.of("10.10.10.1", "10.10.10.1", null),
             Arguments.of("10.10.10.1:80", "10.10.10.1", "80"),
-            Arguments.of("[0::0::0::1]", "[0::0::0::1]", null),
-            Arguments.of("[0::0::0::1]:80", "[0::0::0::1]", "80"),
-            Arguments.of("0:1:2:3:4:5:6", "[0:1:2:3:4:5:6]", null),
```

Comment:
All of the above examples were moved from "valid" to "invalid" as they are not seen as valid by Java itself.

They all fail in `InetAddress`, and `URI` usages for various reasons (see `invalidAuthorityProvider` comments for specific reasons)